pkg/cmd/flags/dnscache.go

```diff
@@ -0,0 +1,75 @@
+package flags
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+
+	"github.com/aquasecurity/tracee/pkg/dnscache"
+)
+
+func dnsCacheHelp() string {
+	return `Select different options for the DNS cache.
+
+Example:
+  --dnscache enable  | enable with default values (see below). Using other options will override them, but is redundant.
+  --dnscache size=X  | will cache up to X dns and ip cross references in the LRU cache (default: 500).
+  --dnscache ttl=Y   | dns and ip entries will expire after Y seconds (default: 300 seconds).
```

Comment:
default TTL is usually 3600 or 86400. Why such a short TTL ? My line of thoughts: if we keep the entry more time, and there are multiple IPs for the same entry, then its good to have all of the associated to the same entry. 300 secs is usually used in very dynamic envs when DNS owners want fast propagation, or during changes so they can make sure all is good before raising it, or related to DNS caching (which is not what we are doing, we are caching "information" about DNS, not DNS itself). No ?

Reply:
Well, the LRU shares expiry throughout it's entries. So entries can't have a unique TTL (otherwise I could take it from the answer). By this logic we should aim at the ceiling for "overall" TTL (so probably 86400 should be chosen), since if an entry had short TTL, the node would probably have to requery it anyway, thus overriding it in the cache.

Reply:
See above.